src/components/CanvasSaveButton/CanvasSaveButton.js

```diff
@@ -206,22 +206,81 @@ class CanvasSaveButton extends PureComponent {
     mapToExport.setTarget(null);
   }
 
-  drawCopyright(destContext, clip) {
+  // Ensure the font size fita with the image width.
+  decreaseFontSize(destContext, arrowWidth, copyright, scale) {
+    const minFontSize = 8;
+    let sizeMatch = destContext.font.match(/[0-9]+(?:\.[0-9]+)?(px)/i);
```

Comment:
you already do it in the `do`
```suggestion
    let sizeMatch;
```